scene/debugger/scene_debugger.h

```diff
@@ -235,7 +235,9 @@ class RuntimeNodeSelect : public Object {
 	enum NodeType {
 		NODE_TYPE_NONE,
 		NODE_TYPE_2D,
+#ifndef _3D_DISABLED
 		NODE_TYPE_3D,
+#endif // _3D_DISABLED
```

Comment:
If I recall correctly, the policy is to keep enumerated values like this in existence, but then just make them act as dummy values when using them. For example, see the `Monitor` enum in `main/performance.cpp`. This code appears to already compile fine without this change, since `scene_debugger.cpp` already had `#ifndef _3D_DISABLED`, you just reordered the lines.